packages/react-core-notifications/.estlintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extends: ['amplify-ui/react'],
```

Comment:
Looks like this need to be updated to use the scoped namespace (same with line 8)

Reply:
Oh yeah
Updated in the later PRs 
Will fix here too